### PR TITLE
Updated fetch,md

### PR DESCRIPTION
**Pull request by rahulgithub-web**
Replaced normal function to an arrow function in line no. 42 and a specific status code 500 (which is used for Internal Server Error) in catch in line number 60 with the status false and with a proper message whatever comes from the server side.

### CODE REVIEW

File: docs/source/packages/fetch.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ async function postData (url, data) {
       const data = await response.json();
       return response(null, data);
     } catch (err) {
-      return response(err, null);
+      return res.status(500).send({status: false, msg: err.message});
```
Comment (StorytellerCZ) [inline docs/source/packages/fetch.md:60]: This doesn't look correct. Have you tried to run this sample? Where is `res` coming from?

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: docs/source/packages/fetch.md (hunk 1 above)
Comment (rahulgithub-web) [reply docs/source/packages/fetch.md:60]: res is same like response

File: docs/source/packages/fetch.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ For the most part though, you will only need to import `fetch` and `Headers`.
 import { Meteor } from 'meteor/meteor';
 import { fetch, Headers } from 'meteor/fetch';
 
-async function postData (url, data) {
+const postData =async(url, data) => {
```
[the remark this answers is not in the document]
Comment (rahulgithub-web) [reply docs/source/packages/fetch.md]: ok sir done

**Comment by rahulgithub-web** [comment]
Done sir, added space after the equal to sign(=) and in line 60 res should be like response.

File: docs/source/packages/fetch.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ async function postData (url, data) {
       const data = await response.json();
       return response(null, data);
     } catch (err) {
-      return response(err, null);
+      return response.status(500).send({status: false, msg: err.message});
```
Comment (StorytellerCZ) [inline docs/source/packages/fetch.md:60]: I do not recall the response function having a status or send methods. This looks like Express thing.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: docs/source/packages/fetch.md (hunk 3 above)
Comment (rahulgithub-web) [reply docs/source/packages/fetch.md:60]: Yeah sir, you are right. I will look forward to it sir. Thank you.

**Pull request closed without merging**